ficlatte/settings-example.py

```diff
@@ -1,106 +0,0 @@
-"""
-Django settings for ficlatte project.
```

Comment:
Settings.py is under the .gitignore file so it won't ever be added to the commit.